src/main/js/src/store/actions/user.js

```diff
@@ -26,10 +27,11 @@ const fetchUserSuccess = identity => {
   if (identity) {
     let isAdmin = false;
     if (identity.organizations) {
-      const organization = identity.organizations[0];
-      isAdmin = organization.permissions.some(
-        p => p.palvelu === 'YKI' && p.oikeus === 'YLLAPITAJA',
-      );
+      isAdmin = identity.organizations.some(({ oid, permissions}) =>
```

Comment:
Tää tais olla korjaus jonka Salla pyysi kun virkailijana useita organisaatioita? Näyttää anyway toimivalta

Reply:
Jep, juuri näin.